services/121-service/test/helpers/utility.helper.ts

```diff
@@ -103,6 +103,13 @@ export async function removeProgramAssignment(
     .send();
 }
 
+export async function runCronJobDoNedbankReconciliation(): Promise<void> {
```

Comment:
Does this still have value with introducing the cronjobs integration test?

Reply:
Yes. It's used frequently to test the reconciliation flow in our integration tests